pokedex_async_redux/lib/features/pokemon_overview/widgets/pokemon_card.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_asyn_redux/utils/constants.dart';
+import 'package:pokedex_asyn_redux/utils/string_extensions.dart';
+
+class PokemonCard extends StatelessWidget {
+  const PokemonCard({
+    required this.pokemon,
+    required this.imageURL,
+    Key? key,
+  }) : super(key: key);
+
+  final String imageURL;
+  final String pokemon;
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      margin: const EdgeInsets.all(cardDefaultSize),
```

Comment:
the thing is u named ur constant as `cardDefaultSize`, yet you're using it for margin which is a bit misleading 😝